modelci/types/proto/service_pb2.py

```diff
@@ -0,0 +1,156 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: modelci/types/proto/service.proto
+
+import sys
+
+_b = sys.version_info[0] < 3 and (lambda x: x) or (lambda x: x.encode('latin1'))
+from google.protobuf import descriptor as _descriptor
+from google.protobuf import message as _message
+from google.protobuf import reflection as _reflection
+from google.protobuf import symbol_database as _symbol_database
+
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+DESCRIPTOR = _descriptor.FileDescriptor(
+    name='modelci/types/proto/service.proto',
+    package='',
+    syntax='proto3',
+    serialized_options=None,
+    serialized_pb=_b(
+        '\n!modelci/types/proto/service.proto\"C\n\x0cInferRequest\x12\x12\n\nmodel_name\x18\x01 \x01(\t\x12\x0c\n\x04meta\x18\x02 \x01(\t\x12\x11\n\traw_input\x18\x03 \x03(\x0c\"+\n\rInferResponse\x12\x0c\n\x04json\x18\x01 \x01(\t\x12\x0c\n\x04meta\x18\x02 \x01(\t2g\n\x07Predict\x12(\n\x05Infer\x12\r.InferRequest\x1a\x0e.InferResponse\"\x00\x12\x32\n\x0bStreamInfer\x12\r.InferRequest\x1a\x0e.InferResponse\"\x00(\x01\x30\x01\x62\x06proto3')
+)
+
+_INFERREQUEST = _descriptor.Descriptor(
+    name='InferRequest',
+    full_name='InferRequest',
+    filename=None,
+    file=DESCRIPTOR,
+    containing_type=None,
+    fields=[
+        _descriptor.FieldDescriptor(
+            name='model_name', full_name='InferRequest.model_name', index=0,
+            number=1, type=9, cpp_type=9, label=1,
```

Comment:
Are those two grpcio generated files related to the type of OS?

Reply:
I tested on Windows, with the same version of grpcio-tools. It has slight differences. I think those difference are not important:
`service_pb2.py`
![image](https://user-images.githubusercontent.com/36268431/94361124-9a86c880-00e4-11eb-8f22-9777d27042ae.png)
A new kwargs provided. Missing of this args only create a warning.

`service_pb2_grpc.py`
![image](https://user-images.githubusercontent.com/36268431/94361220-5942e880-00e5-11eb-8ad2-9166f72e862b.png)
A new class related to external API. Not sure what will this do, we do not use this class anywhere.